ui_tests/src/test/resources/com/saucedemo/account/signinPositive.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Sign In
+
+  @signIn
+  Scenario: Valid user sign in
+    Given I open start page
+    When I set 'standard_user' as username on Sign in page
+    And I set 'secret_sauce' as password on Sign in page
+    And I click 'Sign In' button on Sign in page
+    Then I am on Products page
+
+  @signIn
+  Scenario: Valid user sign in
+    Given I open start page
+    When I set 'locked_out_user' as username on Sign in page
+    And I set 'secret_sauce' as password on Sign in page
+    And I click 'Sign In' button on Sign in page
+    Then I am on Products page
```

Comment:
Locked out user `locked_out_user ` is not entitled to access the application, i.e. trying to sign in as this user is `negative` test case (not positive). In `signInPositive.feature` we're supposed to have only positive test cases